python/examples/core/1-first-steps.py

```diff
@@ -0,0 +1,274 @@
+"""
+.. _core-tutorial-first-steps:
+
+First steps with metatensor
+===========================
+
+.. |CO2| replace:: CO\\ :sub:`2`
+
+This tutorial explores how data is stored inside metatensor's ``TensorMap``, and how to
+access the associated metadata. This is a companion to the :ref:`core classes overview
+<core-classes-overview>` page of this documentation, presenting the same concepts with
+code examples.
+
+To this end, we will need some data in metatensor format, which for the sake of
+simplicity will be loaded from a file. The code used to generate this file can be found
+below:
+
+.. details:: Show the code used to generate the :download:`spherical-expansion.npz`
+             file, or use the link to download it
+
+    ..
+
+        The data was generated with `rascaline`_, a package to compute atomistic
+        representations for machine learning applications.
+
+        .. _rascaline: https://luthaf.fr/rascaline/latest/index.html
+
+        .. literalinclude:: spherical-expansion.py.example
+            :language: python
+
+The :py:class:`TensorMap` stored in the file contains a machine learning representation
+(the spherical expansion) of all the atoms in a |CO2| molecule. You don't need to know
+anything the spherical expansion to follow this tutorial!
+
+.. py:currentmodule:: metatensor
+
+"""
+
```

Comment:
Should we maybe add the standard sentence that we start by important this and that. If you don't want to at least we should add an empty "cell delimiter" 

```python
# %%
#
 ```
 
 Then editors like vs code also see the import section as cell, otherwise you can't execute them easily.